src/v/cluster/rm_stm.cc

```diff
@@ -2067,6 +2067,12 @@ ss::future<stm_snapshot> rm_stm::take_snapshot() {
     }
     _log_state.abort_indexes = std::move(abort_indexes);
 
+    vlog(
+      _ctx_log.debug,
+      "Removing abort indexes {} with offset < {}",
```

Comment:
if you are talking about a numeral, I would write
```
"Removing {} abort indexes with offsets < {}"
```

Reply:
Just want to avoid another round of CI, will incorporate this suggestion in the actual fix that fixes the test.